index.html

```diff
@@ -514,43 +523,66 @@
         </dt>
       </dl>
       <p>
-        A <code>PermissionStatus</code> instance has an associated
-        <var>permission</var> that is a <a>PermissionDescriptor</a>.
+        <code>PermissionStatus</code> instances are created with the following
```

Comment:
`PermissionStatus`es are going to have two internal slots once we get to #66, in case that makes it look a little better: https://rawgit.com/jyasskin/permissions/allow-choosers/index.html#idl-def-PermissionStatus.

I'm finding in [Web Bluetooth](https://webbluetoothcg.github.io/web-bluetooth/#bluetoothservicedatamap) that having a bunch of internal slots makes it easier to understand the purpose of each bit of associated data, and that `foo@[[bar]]` is simpler to type and link than "the bar associated with `foo`". The `PermissionStatus` slots are simple enough that I can eliminate them without needed a lot of extra wording, but I do somewhat prefer them.

@annevk and @domenic may also have opinions about which to use.